server/driver/wivrn_foveation.cpp

```diff
@@ -142,10 +142,33 @@ static std::tuple<float, float> solve_foveation(float λ, float c)
 	return {a, b(a)};
 }
 
+static float get_angle_offset()
+{
+	if (const char * cvar = std::getenv("WIVRN_FOVEATION_OFFSET"))
+	{
+		std::string_view var(cvar);
+		float res;
+		auto e = std::from_chars(var.begin(), var.end(), res);
+		if (e.ec == std::errc())
+			return -res * M_PI / 180;
+		else
+			U_LOG_W("Malformed WIVRN_FOVEATION_OFFSET, must be a number (angle in °)");
+	}
+	return 0;
+}
+
+static bool is_zero_quat(xrt_quat q)
+{
+	return q.x == 0 and q.y == 0 and q.z == 0 and q.w == 0;
+}
+
 static xrt_vec2 yaw_pitch(xrt_quat q)
 {
-	if (q.x == 0 and q.y == 0 and q.z == 0 and q.w == 0)
-		return xrt_vec2{};
+	if (is_zero_quat(q))
+	{
+		// Add manual offset, useful if the default angle does not suit the user
+		return {0, get_angle_offset()};
+	}
```

Comment:
yaw_pitch is used with both view and gaze. i think let's pass a default value as arg instead